blog/_posts/2016-09-16-juliacon2016.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: JuliaCon 2016
+author: Ranjan Anantharaman
+---
+
+The gentle breeze brushed my face and the mild sunshine warmed an
+otherwise chilly morning. I was standing in front of a large building that can
+only be described as unique: a series of metal plates jutting out at odd angles,
+whose dull resplendence cast an instant impression. It was the Ray and Maria
+Stata Centre, a towering monolith which, the venue for an event that
+people from all over the world came to attend and participate in. I was in town for
+the [third edition of JuliaCon](https://www.youtube.com/watch?v=EZD3Scuv02g&list=PLP8iPy9hna6SQPwZUDtAM59-wPzCPyD_S),
+the annual Julia Conference at MIT.
+
+On the eve of JuliaCon, and a series of workshops were organised on some important
+areas people use Julia for. I was conducting the
+[Parallel Computing workshop](https://www.youtube.com/watch?v=euZkvgx0fG8)
+along with some other members of the JuliaLab. The key idea in our workshop was
```

Comment:
Yep, I've added it. Usually outdated diffs aren't shown. I guess this Reviews feature is different.